src/pages/MuteerBeleidsmodulesOverview/TableDataCell/TableDataCell.tsx

```diff
@@ -1,14 +1,18 @@
+interface TableDataCell {
```

Comment:
@Aidenbuis If you import FC from React and use it like this:

`const TableDataCell: FC<TableDataCell> = ({ className = '', children }) => (`

you don't need to define the children prop :)